### PR TITLE
fix: set session sample rate configurable and defaults to 0

### PR DESCRIPTION
**Pull request by abdullahwaheed**

### CODE REVIEW

File: src/DatadogLoggingService.js [hunk 1]
```diff
@@ -35,8 +35,8 @@ class DatadogLoggingService extends NewRelicLoggingService {
       service: process.env.DATADOG_SERVICE,
       env: process.env.DATADOG_ENV,
       version: process.env.DATADOG_VERSION,
-      sessionSampleRate: 50,
-      sessionReplaySampleRate: 100,
+      sessionSampleRate: parseInt(process.env.SESSION_SAMPLE_RATE || 0, 10),
```
Comment (robrap) [inline src/DatadogLoggingService.js:38]: Should the 0 be the default? Should that be 50?

File: src/DatadogLoggingService.js [hunk 2]
```diff
@@ -35,8 +35,8 @@ class DatadogLoggingService extends NewRelicLoggingService {
...
+      sessionReplaySampleRate: parseInt(process.env.SESSION_REPLAY_SAMPLE_RATE || 0, 10),
```
Comment (robrap) [inline src/DatadogLoggingService.js:39]: 0 default seems right for this for now. Should we add a comment that we are defaulting to 0 until we work out privacy concerns?

File: src/DatadogLoggingService.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ class DatadogLoggingService extends NewRelicLoggingService {
       site: process.env.DATADOG_SITE,
       env: process.env.DATADOG_ENV,
       forwardErrorsToLogs: true,
-      sessionSampleRate: 100,
+      sessionSampleRate: parseInt(process.env.SESSION_SAMPLE_RATE || 0, 10),
```
Comment (robrap) [inline src/DatadogLoggingService.js:50]: Do we want 100 as our default for now?

File: src/DatadogLoggingService.js (hunk 1 above)
Comment (abdullahwaheed) [reply src/DatadogLoggingService.js:38]: do we want `sessionSampleRate` default  to 50 for all MFEs? if yes then i'll update it

File: src/DatadogLoggingService.js (hunk 3 above)
Comment (abdullahwaheed) [reply src/DatadogLoggingService.js:50]: i was thinking to default everything to 0 and then configure from envs. but if we want default values set, i'll update it

Comment (abdullahwaheed) [reply src/DatadogLoggingService.js:50]: should `sessionSampleRate` for both RUM and Browser logging differ?

i am currently using same env for both

**Review by robrap — APPROVED** [review]
I changed my mind. 0 will work, with explicit settings for each MFE.

**Pull request merged**